.github/workflows/test.yaml

```diff
@@ -55,6 +55,7 @@ jobs:
                   coverage: pcov
                   tools: composer:v2
 
+
```

Comment:
More spaces than needed @wbloszyk @VincentLanglet .

Reply:
https://github.com/sonata-project/dev-kit/pull/1455

But I try to fail the ci with two lines without success: https://github.com/sonata-project/SonataAdminBundle/pull/6925/files